services/worker_service.rb

```diff
@@ -47,6 +51,32 @@ def start_workers_for_project_and_credential(project: nil, provider_credential:
       self.project_to_workers_dictionary[workers_key] = new_workers
     end
 
+    def start_github_workers
+      return unless Services.onboarding_service.correct_setup?
+      launch_workers unless ENV["FASTLANE_CI_SKIP_WORKER_LAUNCH"]
+    end
+
+    def launch_workers
+      logger.debug("Starting workers to monitor projects")
+      # Iterate through all provider credentials and their projects and start a worker for each project
+      ci_user.provider_credentials.each do |provider_credential|
+        projects = Services.config_service.projects(provider_credential: provider_credential)
```

Comment:
More services calling into a global service.
I think we need to find a way to coordinate this in a less global way. 
Either through coordinator usage, or dependency injection, or something else.
The `launch.rb` was doing all this stuff because it was a one-time setup, in one place, and IMO that's ok for this code smell stuff, as long as all the smell comes from 1 file, that's ok.